packages/edit-post/src/store/test/actions.js

```diff
@@ -129,4 +136,158 @@ describe( 'actions', () => {
 			expect( notices[ 0 ].content ).toMatch( 'template' );
 		} );
 	} );
+
+	describe( 'hideBlockTypes', () => {
+		it( 'adds the hidden block type to the preferences', () => {
+			registry
+				.dispatch( editPostStore )
+				.hideBlockTypes( [ 'core/quote', 'core/table' ] );
+
+			const expected = [ 'core/quote', 'core/table' ];
+
+			// TODO - remove once `getPreference` is deprecated.
+			expect(
+				registry
+					.select( editPostStore )
+					.getPreference( 'hiddenBlockTypes' )
+			).toEqual( expected );
+
+			expect(
+				registry.select( editPostStore ).getHiddenBlockTypes()
+			).toEqual( expected );
+		} );
+	} );
+
+	describe( 'showBlockTypes', () => {
+		it( 'removes the hidden block type from the preferences', () => {
+			registry
+				.dispatch( editPostStore )
+				.hideBlockTypes( [ 'core/quote', 'core/table' ] );
+
+			const expectedA = [ 'core/quote', 'core/table' ];
+
+			// TODO - remove once `getPreference` is deprecated.
+			expect(
+				registry
+					.select( editPostStore )
+					.getPreference( 'hiddenBlockTypes' )
+			).toEqual( expectedA );
+
+			expect(
+				registry.select( editPostStore ).getHiddenBlockTypes()
+			).toEqual( expectedA );
+
+			registry
+				.dispatch( editPostStore )
+				.showBlockTypes( [ 'core/table' ] );
+
+			const expectedB = [ 'core/quote' ];
+
+			// TODO - remove once `getPreference` is deprecated.
+			expect(
+				registry
+					.select( editPostStore )
+					.getPreference( 'hiddenBlockTypes' )
+			).toEqual( expectedB );
+
+			expect(
+				registry.select( editPostStore ).getHiddenBlockTypes()
+			).toEqual( expectedB );
+		} );
+	} );
+
+	describe( '__experimentalUpdateLocalAutosaveInterval', () => {
```

Comment:
I've added tests for all the other preferences to bring some confidence that the changes to `getPreferences` and `getPreference` were correct. Plus they'll help when I come to migrate the other preferences.